src/Discovery/DiscoveryDetails/DataDownloadList/ActionButtons/DownloadUtils/HandleDownloadManifestClick.ts

```diff
@@ -5,15 +5,15 @@ import DownloadJsonFile from './DownloadJsonFile';
 const HandleDownloadManifestClick = (
   config: DiscoveryConfig,
   selectedResources: any[],
-  healIDPLoginNeeded: boolean,
+  healIDPLoginNeeded: string[],
```

Comment:
Suggestion: Rename variable to ```healIDPLoginInfo``` 

In my opinion the name ```healIDPLoginNeeded``` sounds like the name of something that is either true or false

Reply:
yeah, the true intention behind this variable is "the IDP that the user is required use to login, but has not satisified yet"
I think `healIDPLoginInfo` is a bit too generic, but I also don;t think `healIDPLoginNeeded` is a great name. So still open to suggestions 🤔

Reply:
great suggestion, `missingRequiredIdentityProviders` was the chosen one, I have updated all the variable names